patterns/2-structured/contracted-contributor.md

```diff
@@ -27,12 +27,12 @@ them, the InnerSource initiative is just one of many initiatives to foster
 innovation and efficiency, though. They are funding InnerSource with money and
 capacity for community leaders and are largely giving autonomy as for how the
 budget is spent. They are also limiting the breadth and duration of the
-initiative and partake in periodic reviews until there is proof that it yields
+initiative and partake in periodic reviews until there is proof it yields
 the expected results (see [Review Committee](review-committee.md)).  Top level
 management has announced their support for InnerSource on various company
 internal meetings.
 
-However, top level management has not yet empowered or incentivised mid-level
+However, top-level management has not yet empowered or incentivized mid-level
```

Comment:
```suggestion
However, top-level management has not yet empowered or incentivised mid-level
```

We tend to go with the American English spelling, so I am changing this back to "incentivised".